tools/cli/admin_elastic_search_commands.go

```diff
@@ -456,14 +463,14 @@ func generateCSVReport(reportFileName string, headers []string, tableData [][]st
 	if err != nil {
 		fmt.Printf("Error write to file, err: %v", err)
 	}
-	f.Close()
+	return f.Close()
```

Comment:
I _think_ this will also return the writestring err, if there is one - if so, we could probably just get rid of that earlier `if err { print }` branch.